src/probnum/linops/_linear_operator.py

```diff
@@ -215,28 +318,56 @@ def cond(self, p=None):
         cond : {float, inf}
```

Comment:
Is this return type in the docstring unnecessary, due to automatic documentation of type hints?